src/main/java/akademia/services/PlanetService.java

```diff
@@ -10,4 +14,9 @@
     public PlanetService(PlanetRepository planetRepository) {
         this.planetRepository = planetRepository;
     }
+
+    @GetMapping("/planets")
```

Comment:
to nie powinno tu byc w tylko w kontrollerze. blad w wymaganiach, zaraz ja poprawie